### DISCUSSION TITLE
Failure to lookup LDAP

### DISCUSSION THREAD

**Discussion by mailliwal** [lldap/lldap#741] [Q&A]
Hi,

I can connect LDAP with "admin" with failure to looj-up & test user credential.

cn:    cn=lldap_strict_readonly,ou=groups,dc=test,dc=home

login:      cn=admin,ou=people,dc=test,dc=home

May I know CN is incorrect or other issue ?

Thanks

**Comment by nitnelave** [comment]
Hi! How are you connecting? On the web UI? Then the login would just be "admin". With an LDAP client? Then the "cn=admin,..." (or better: "uid=admin,...") is correct

**Reply by mailliwal** [reply]
Account could be looked-up with below configuration.

- CN = uid
- DN = dc=example,dc=com
- username = cn=admin,ou=people,dc=example,dc=com

If changed DN to dc=lldap_strict_readonly,ou=groups,dc=example,dc=com

Then authentication failure.

May I know it's correct ?

THANKS

**Reply by nitnelave** [reply]
You can't log in as a group (lldap_strict_readonly), only as a user.

How are you connecting? Are you using a service that has an LDAP backend? Are you using `ldapsearch`? Can you provide more details? I.e. the specific command that was run to get the failure, or the entire service configuration and/or LLDAP configuration.

You can also head to Discord if you want where we can talk about it more dynamically.
